interview.md

```diff
@@ -5,47 +5,75 @@
 experience like now? What would their experience be like if you could help
 them?_
 
+- Quality of Life in Psychology Research. I am helping research assistants and anyone doing the more monotonous work of research. Ideally the issues during data processing would shrink, data could get processed faster and life would be easier lmao. Right now, it is very frustrating. 
+- Lynnette uses EEG for alot of the research, so when software crashes or slows, it also wastes the time with the subject of the research. 
+-
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+- I this isnt really the project we are being assigned, but since this is just an idea, I think its fine to break from the heard a little. I think its good because, if I can better the GUI of a DSL I can hopefully prevent RA's from hating them. I want to improve existing resources rather then ignore thme. 
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+- I just felt like it was the most reasonable way (as a project) to address lynnettes concerns. We don't need NEW languages we need to improve what we have see: 
+https://xkcd.com/927/
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+- Psychology research and data anaysis 
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+- Ironically we want to remove coding as much as possible. Ideally keeping simple clear and concise (GUI's) 
+- This does remove some technical aspects, but for the mundane data analysis we dont need the strongest possible tool.
+
 ### Operation (semantics)
 
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+- We will have to see! I imagine it will be much more clear concise and quick. We want automatic previews like that one video. 
+- Errors are SO important to report in research so that will be the priority
+
 ### Expressiveness
 
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+- Easy: Mundane quick repetitive copy and paste work 
+- The complex data analysis 
+
 ### Related work
 
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+ - There are quite a few, in fact i need to find one to "improve". In reaality this is more of a thought expiriment of a project. 
+ - I want to see what does work and what doesn't work in this specific domain. I want to see why shes so frustrated.
```

Comment:
I feel like a lot of this project would be research if you don't already know a lot about psychology.